Jenkins build failed with signature failure

Build fails with:

```
[INFO] [INFO] --- maven-gpg-plugin:1.6:sign (sign-artifacts) @ microprofile-health-parent ---
[INFO] gpg: signing failed: Not a tty
[INFO] gpg: signing failed: Not a tty
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Reactor Summary for MicroProfile Health 2.2-RC1:
[INFO] [INFO] 
[INFO] [INFO] MicroProfile Health ................................ FAILURE [ 35.174 s]
[INFO] [INFO] MicroProfile Health API ............................ SKIPPED
[INFO] [INFO] MicroProfile Health TCK ............................ SKIPPED
[INFO] [INFO] MicroProfile Health Spec ........................... SKIPPED
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD FAILURE
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Total time:  58.561 s
[INFO] [INFO] Finished at: 2020-01-22T10:59:12Z
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.6:sign (sign-artifacts) on project microprofile-health-parent: Exit code: 2 -> [Help 1]
```

--- a/pom.xml
+++ b/pom.xml
@@ -380,6 +380,12 @@
                                 <goals>
                                     <goal>sign</goal>
                                 </goals>
+                                <configuration>
+                                    <gpgArguments>
+                                        <arg>--pinentry-mode</arg>
+                                        <arg>loopback</arg>
+                                    </gpgArguments>
+                                </configuration>
                             </execution>
                         </executions>
                     </plugin>

